test/e2e-tests/README.md

```diff
@@ -49,5 +55,7 @@ Set the go binary used for testing that is in your `PATH`
 make GO=go1.19 go-test
 ```
 
+If you want to re-run the tests in the same namespace you can just re-run the `go-test` target. The tests fail if there are **any** failed pipeline runs. Therefore if you have a failed run and want to re-test make sure to delete any that have failed.
+
```

Comment:
Maybe we could add an option to create a namespace as well? Or make namespace creation a part of the `go-test-setup`  target.

Reply:
Also because of the changes to make setup use `oc apply` instead of `oc create` it means you have less of a chance to want to delete the namespace.

My guess is that making the tests ignore failed pipeline runs if they failed before it started would be the thing to do here

Reply:
I meant that both delete and create could be automatic as we do it here:
https://github.com/opendatahub-io/ai-edge/blob/7d76dedd31d8410f03b5b37bc715daeae8c6fc6a/test/shell-pipeline-tests/openvino-tensorflow-housing/pipelines-test-openvino-tensorflow-housing.sh#L8-L9
But I think we can decide on that later.